util/exercise/src/main.rs

```diff
@@ -102,7 +102,7 @@ fn process_matches(matches: &ArgMatches<'_>) -> Result<()> {
         }
 
         ("", None) => {
-            println!("No subcommand was used.\nUse init_exercise --help to learn about the possible subcommands.");
+            println!("No subcommand was used.\nUse exercise --help to learn about the possible subcommands.");
```

Comment:
This change makes sense, but seems pretty unrelated to the rest of this PR.

Reply:
I agree. How do you feel about documenting something like a Unix philosophy for our pull requests?

Keep it small.
One logical change or something.
I can take a stab for #1111.

Reply:
Yeah, "one logical change" is the best expression of what I want to see in a PR. It's not necessarily small--there have been good, useful PRs which added hundreds of lines of code and took months to resolve (looking at you, #653)--but life is easier when PRs are focused.